tests/Unit/Console/UpgradeDatabaseTest.php

```diff
@@ -52,10 +52,8 @@ public function testV2Database()
         $this->artisan('migrate:fresh');
         $migrations_v4 = DB::table('migrations')->pluck('migration')->toArray();
 
-        // Check migration table after upgrade starts with the same as a fresh v4 database
-        foreach ($migrations_upgrade as $key => $migration) {
-            $this->assertEquals($migration, $migrations_v4[$key]);
-        }
+        // Check migration table after upgrade looks like a fresh v4 database
```

Comment:
Consider adding a clause:'Check the migration table after the upgrade to see if it looks like a new v4 database.'